include/ck/tensor_operation/gpu/grid/gridwise_gemm_pipeline_selector.hpp

```diff
@@ -5,6 +5,7 @@
 
 #include "ck/tensor_operation/gpu/grid/gridwise_gemm_pipeline_v1.hpp"
 #include "ck/tensor_operation/gpu/grid/gridwise_gemm_pipeline_v2.hpp"
+#include <iostream>
```

Comment:
Is this needed in kernel code?

Reply:
I'm concerned with host library dependency being added into device code.

Pin myself @asroy here for further review

Reply:
> I'm concerned with host library dependency being added into device code.

We arent adding a host library dependency, this device function already depends on host code. We are just fixing a compilation error. Removing the host dependency is really beyond the scope of this PR.